File: Programming/Python/00/0001 Grading.py
# Input work, midterm and final score
work = int(input())
midterm = int(input())
final = int(input())

# Calculate the final score
score = work + midterm + final

# Grading
# A  80-100
# B+ 75-79
# B  70-74
# C+ 65-69
# C  60-64
# D+ 55-59
# D  50-54
# F  0-49
if(score>=80 and score<=100):
    print("A")
elif(score>=75 and score<=79):
    print("B+")
elif(score>=70 and score<=74):
    print("B")
elif(score>=65 and score<=69):
    print("C+")
elif(score>=60 and score<=64):
    print("C")
elif(score>=55 and score<=59):
    print("D+")
elif(score>=50 and score<=54):
    print("D")
else:
    print("F")

File: Programming/C++/00/0001 Grading.cpp
#include<bits/stdc++.h>
using namespace std;
int work, midterm, final, score;

int main ()
{
    // Input work, midterm and final score
    cin >> work;
    cin >> midterm;
    cin >> final;
    
    // Calculate the final score
    score = work + midterm + final;

    // Grading
    // A  80-100
    // B+ 75-79
    // B  70-74
    // C+ 65-69
    // C  60-64
    // D+ 55-59
    // D  50-54
    // F  0-49
    if(score>=80 && score<=100)
    {
        cout << "A";
    }
    else if(score>=75 && score<=79)
    {
        cout << "B+";
    }
    else if(score>=70 && score<=74)
    {
        cout << "B";
    }
    else if(score>=65 && score<=69)
    {
        cout << "C+";
    }
    else if(score>=60 && score<=64)
    {
        cout << "C";
    }
    else if(score>=55 && score<=59)
    {
        cout << "D+";
    }
    else if(score>=50 && score<=54)
    {
        cout << "D";
    }
    else
    {
        cout << "F";
    }

    //End of the program
    return 0;
}